Create ipad-requires-fullscreen.md (#38)
expo/fyi: ipad-requires-fullscreen.md

# iPad requires full screen

> When submitting your app to App Store, you may encounter an error saying that your app orientation doesn't support iPad multitasking. This happens, because Expo apps do not support multitasking by default and, in that case, your app has to be open in fullscreen. 

Reviewer: ```suggestion
When submitting your app to App Store, you may encounter an error saying that your app orientation doesn't support iPad multitasking. This happens because Expo apps do not support multitasking by default and, when multitasking isn't supported, Apple requires that your app is opened in fullscreen.
```
